src/test/java/jenkins/bouncycastle/EncodignDecodingTest.java

```diff
@@ -224,5 +226,15 @@ public void testCreationFromObjectPrivateKeyPEM() throws Exception {
 
         assertEquals(FileUtils.readFileToString(PRIVATE_KEY_PEM), FileUtils.readFileToString(pemFileNew));
     }
+    
+    
+    @Test
+    @Issue(value="JENKINS-35661") 
+    public void testReadKeyPairFromPCKS8PEM() throws Exception {
+        PEMEncodable pemEnc = PEMEncodable.read(getResourceFile("private-key-pcks8.pem"));
+        assertNotNull(pemEnc.toKeyPair());
+        assertNotNull(pemEnc.toPrivateKey());
+        assertNotNull(pemEnc.toPublicKey());
```

Comment:
@stephenc checked. In our case we cannot make any assumption about the PublicExponent being used... 

I've debugged the test and the Public exponent is, as you predicted, 65537. 

do you have any other test in mind or approach to get the Public Exponent? should I force a key with a different PublicExponent just to make sure?